src/ast.rs

```diff
@@ -370,12 +368,32 @@ impl PartialOrd for RawBinaryOperator {
     }
 }
 
+/// *Part* of a let binding: the variable declaration and value it is bound to. What is missing
+/// is the expression in which the bound name is in-scope.
```

Comment:
I may not be parsing this right; a let binding needs to have a sort-of upreference to its enclosing scope?

Reply:
Ah; I understand after reading below. Makes sense, I wasn't reading precisely. IDK why I'm not just deleting this comment chain, then.